projects/netgrif-components-core/src/lib/data-fields/multichoice-field/multichoice-autocomplete-field/abstract-multichoice-autocomplete-field-component.component.ts

```diff
@@ -83,4 +83,8 @@ export abstract class AbstractMultichoiceAutocompleteFieldComponentComponent imp
         }
         return key;
     }
+
+    public getValueFromKey(key: string) {
+        return this.multichoiceField.choices.find(choice => choice.key === key).value;
```

Comment:
If no choice with the provided key is found this statement will throw an error. Please replace with operator `?.` after the find function and add the return type to the method.

Reply:
Done